server/middleware/error_handler.js

```diff
@@ -53,16 +53,15 @@ module.exports = (err, req, res, next) => {
     err.code = err.status || code;
   }
 
-  // We don't pollute the tests results when running mocha unless the DEBUG variable is set
-  if(process.env.DEBUG || !process.argv[1].match(/\/mocha\//)) {
-
-    if(req.app.set('env') === 'production') {
+  switch (process.env.NODE_ENV) {
+    case 'production':
       sendErrorByEmail(req, err);
-    }
-
-    console.error('Error Express : ', err);
-    if (err.stack) console.log(err.stack);
+      break;
+    case 'development':
```

Comment:
The stack then isn't printed, as was already the case whenever `DEBUG` wasn't defined.